databases/week1/data.sql

```diff
@@ -0,0 +1,58 @@
+--Find out how many tasks are in the task table
+
+SELECT * FROM TASK;
```

Comment:
You do not need this statement i.e. to select all the records in the tasks table

Reply:
Thanks for your feedback